src/main/java/gregtech/common/metatileentities/multi/multiblockpart/MetaTileEntityMaintenanceHatch.java

```diff
@@ -429,6 +428,7 @@ public void readFromNBT(NBTTagCompound data) {
         // Legacy Inventory Handler Support
```

Comment:
How much legacy this is and can it be thrown away?

Reply:
This is the old inventory contents stored in the NBT of the tile. It is legacy from the previous PR which transitioned to using a plain `ItemStackHandler` instead of the combined input and output inventories. 

It is safe to discard the NBT here, as the Inventory contents have already been read via NBT into the new system, and discarding the NBT here prevents this check from being hit again.

Reply:
I see it is from 09.03.2023 so it is not legacy enough to throw it away.

Reply:
It is not being thrown away. The existing data is being converted into the new format, and then the key is being removed so that it does not attempt to convert into the new format again.

Reply:
No, you miss understand me. I was thinking about having you to throw this whole legacy part away. As everyone should be converted already, but it is not legacy enough for it to be the case.